src/components/Slider/Slider.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RCSlider from 'rc-slider/lib/Slider';
+import './Slider.scss';
+
+const Slider = React.forwardRef((props, ref) => <RCSlider {...props} ref={ref} />);
```

Comment:
@mukiienko to make this component controllable by ref passed in props

Reply:
For me not clear the reason make it controlled through the ref. Seems  `RCSlider` have a bunch of props for managing itself. Could you explain your point, please?

Reply:
@mukiienko this functionality is a part of the design document

Reply:
I think we suppose to use a native `range input` for the Slider. In this case, `ref` makes sense :)

Reply:
@mukiienko unfortunately native component can't be used here because of the styling limitations and cross-browser differentiations. You can read about them [here](https://css-tricks.com/sliding-nightmare-understanding-range-input/).
Ref is passed to the component from the outside because it's still technically an input, so it might be useful to have a possibility to get its props, state, values and update them at some point.